Global/Eclipse.gitignore

```diff
@@ -49,3 +49,15 @@ local.properties
 
 # Code Recommenders
 .recommenders/
+
+# Java class files
+**/*.class
+**/classes
+**/classes/**
+
+# svn plugin
+**/.svn
```

Comment:
Yeah,may be we'd better make a new gitignore file for Subversion plugin or put them together.






At 2016-05-01 16:59:49, "Brendan Forster" <notifications@github.com> wrote:


In Global/Eclipse.gitignore:

> @@ -49,3 +49,15 @@ local.properties
>  
>  # Code Recommenders
>  .recommenders/
> +
> +# Java class files
> +**/*.class
> +**/classes
> +**/classes/**
> +
> +# svn plugin
> +**/.svn


We have a precedence here around externalizing other VCSes in their own templates - for example Mercurial.gitignore.

We don't have one for Subversion though - do you think that's a suitable compromise here?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub